### ISSUE TITLE
Requirements - public health official needs are not captured

### ISSUE THREAD

**Issue by peterkwells** [CrowdNotifier/documents#23]
**Summary** 

The paper does not describe the needs and context of local public health officials, either as user needs or functional/non-functional requirements. This risks impacting public health effectiveness.

**Detail**

In many countries contact tracing is currently performed by local public health officials who liaise with residents, venues, hospitals, enforcement authorities etc to perform their role. CrowdNotifier is a protocol designed to support some of these activities. The paper does not describe public health official's needs or how the protocol, or wider system, would meet them.

For example the non-functional requirements state that the service should be easy for resident to use and for locations to deploy, but has no mention of whether public health officials need a service that is easy to trigger, or what monitoring needs they have to understand if the service is being effective or how that effectiveness varies across the population.

It is important for readers of the paper to be aware that some of those needs may be supported by activities outside the protocol (for example, perhaps measuring effectiveness across differing population groups is met by quantitative surveys, or monitoring spread of the virus is performed through infection surveys and monitoring of hospital statistics). This would be useful discussion for the paper to contain.

Unless public health official needs are reasonably well understood then we risk missing important ones and damaging effectiveness.

**Comment by Thomas-Hendrik** [comment]
Taking into account that the German government has released funds for the digital modernization of the health authorities (Gesundheitsämter), there might be the opportunity to raise money for such a development.

**Comment by Thomas-Hendrik** [comment]
I have just heard, that the health authorities (Gesundheitsämter) in Böblingen use an analysation Software by SAP and in Bayern another Software, provided by the local IT-Service Agency AKDB. Its typical for the dezentralized, local governments and there IT departments...

**Comment by r-r-liu** [comment]
Yes, I agree with @peterkwells.  This seems to ignore some rather basic realities of containing COVID-19.  First, short of lockdowns TTIQ (Testing Tracing Isolating Quarantining) is the most effective way of breaking infection chains.  These are activities that require the force of law, hence are performed by qualified personnel and/or institutions.  Only in an ideal world could all users of an app who are advised to quarantine themselves be trusted to do so.  And even so, what purpose is served by not so informing, indeed, not being able to so inform, public health authorities, who else was in the same closed space as an index person?  Surely, an app based on CrowdNotifier would not meet the requirements of a restaurant to collect guests' contact information for public health authorities.
